tests/testthat/tests-config.R

```diff
@@ -20,3 +20,9 @@ test_that("persisting to config alters environment", {
   expect_equal(gsplot:::config("points")$col, "blue")
   cleanup()
 })
+
+test_that("formals are correctly retrieved", {
+  expect_equal(length(config("par")), 6)
+  expect_equal(names(config("points")), c("pch", "col"))
+  expect_equal(length(config("arrows")), 0)
+})
```

Comment:
Should we add a test for something that's not allowed and see what it should return?